tests/unit/layouts/ModelizerDrawLayout.spec.js

```diff
@@ -134,13 +134,25 @@ describe('Test page component: ModelizerDrawLayout', () => {
     });
 
     it('should do nothing when there is no plugin', async () => {
-      PluginManager.getPluginByName = jest.fn(() => null);
+      const notifySpy = jest.spyOn(Notify, 'create');
```

Comment:
Is this change bring something ? If not, no need to update. better stay with the usual way to do.

Reply:
`PluginManager.getPluginByName = jest.fn(() => null);` contaminates a global object and that is not cleaned up when the test finishes.

Removing the line solved the problem and the test still passes. Reading the code again, I am wondering what really was the purpose of this line if the test passes without it?

Reply:
Anyway I will roll it back as I am moving my code from `ModelizerDrawLayout.vue` to `ModelizerDrawPage.vue`, and also moving the tests accordingly.

Reply:
rolled back